### ISSUE TITLE
Check Web-UI with Screen-Reader

### ISSUE THREAD

**Issue by MacLemon** [anlumo/KaraokeV2#22]
## Describe the bug
The UI should be usable with a screen reader.

### Steps to Reproduce
1. Check with VoiceOver on a Mac or iPhone (because that's what we have at our disposal)

## Notes
We might be lucky finding volunteers to help with testing screen-reader compatibility.

**Comment by MacLemon** [comment]
Notes on Screenreader navigation with VoiceOver on iOS:

Navigating visually from left to right and top to bottom:
- KaraokeParty Heading
- MC Button (should read: MC Login)
- dark/light toggle: Just reads button (should read: light/dark mode toggle)
- Search field: Seems fine with the standard text.
- Language filter reads: Button Globe (should read: filter by langauge)
- Decade filter reads: Button calendar (should read: filter by decade)
- Duett filter reads: Button people (should read something like: filter for solo or duet)
- Explanatory Text cannot be selected in screen reader at all.
- Label: Some suggestions (cannot be selected or read at all)
- Song label (artist/Title) cannot be selected or read at all
- Button to show Cover/Lyrics reads: Button Chat (obviously derived from the icon. Should read: song details.)
- Album cover art: Reads: Show song details (seems OK)
  - Activating that to enter the player: greets with “0 percet”. Focus is on the playhead.
  - Tapping header where song title is shown reads “Dismiss”.
    - Activating that, does *not* actually  dismissthe dialoge.
  - × reads: Button close, can be activated to actually dismiss the dialogue (seems fine)
  - ⏪️ reads: Button (should read: rewind to start)
  - 10 seconds back: reads: Button (should read: rewind 10 seconds)
  - ▶️ reads: “Button play”
    - Can be activated
  - 10 seconds forward: reads: Button (should read: advance 10 seconds)
  - Lyrics: Cannot be selected or read at all
 - song list Search: Cannot be scrolled
 - Song List Browse: Cannot be scrolled
   - Two-finger swipe-down to read an overview stops reading song item buttons after 7 items, though there are more on screen.


Status:
Main use-cases of application cannot be accessed with a screenreader.
- Search for a song
- Get song details
- Submit to the queue
- Get current playlist
